extensions/cornerstone/src/commandsModule.ts

```diff
@@ -415,6 +422,122 @@ const commandsModule = ({ servicesManager }) => {
         (activeViewportIndex - 1 + viewports.length) % viewports.length;
       ViewportGridService.setActiveViewportIndex(nextViewportIndex);
     },
+    activateMPRForActiveViewport: () => {
```

Comment:
Can you call this applyHangingProtocol, taking the name of a HP extension export as the parameter and optionally a stage.